tests/Unit/NumericalAlgorithms/Interpolation/Test_ParallelInterpolator.cpp

```diff
@@ -0,0 +1,382 @@
+struct mock_interpolation_target {
+  using metavariables = Metavariables;
+  using chare_type = ActionTesting::MockArrayChare;
+  using array_index = size_t;
+  using action_list = tmpl::list<>;
+  using component_being_mocked =
+      intrp::InterpolationTarget<Metavariables, InterpolationTargetTag, 3,
+                                 Frame::Inertial>;
+  using initial_databox = db::compute_databox_type<
+      typename intrp::Actions::InitializeInterpolationTarget<
+          InterpolationTargetTag>::template return_tag_list<Metavariables, 3,
+                                                            ::Frame::Inertial>>;
+
+  using const_global_cache_tag_list = Parallel::get_const_global_cache_tags<
+      tmpl::list<intrp::Actions::LineSegment<InterpolationTargetTag, 3,
+                                             Frame::Inertial>>>;
+};
+
+template <typename Metavariables, size_t VolumeDim>
+struct mock_interpolator {
+  using metavariables = Metavariables;
+  using chare_type = ActionTesting::MockArrayChare;
+  using array_index = size_t;
+  using const_global_cache_tag_list = tmpl::list<>;
+  using action_list = tmpl::list<>;
+  using component_being_mocked = intrp::Interpolator<Metavariables, VolumeDim>;
+  using initial_databox =
+      db::compute_databox_type<typename intrp::Actions::InitializeInterpolator<
+          VolumeDim>::template return_tag_list<Metavariables>>;
+};
+
+struct MockMetavariables {
+  struct InterpolationTargetA {
+    using compute_items_on_source = tmpl::list<Tags::SquareComputeItem>;
+    using vars_to_interpolate_to_target = tmpl::list<Tags::Square>;
+    using compute_items_on_target = tmpl::list<>;
+    using compute_target_points =
+        intrp::Actions::LineSegment<InterpolationTargetA, 3, Frame::Inertial>;
+    using post_interpolation_callback =
+        TestFunction<InterpolationTargetA, Tags::Square>;
+    // This tag is also an OptionsTag. The type and help string below
+    // refer to the options that are read from the input file.
+    using type = typename compute_target_points::options_type;
+    static constexpr OptionString help = {"Options for InterpolationTargetA"};
+  };
+  struct InterpolationTargetB {
+    using compute_items_on_source = tmpl::list<Tags::SquareComputeItem>;
+    using vars_to_interpolate_to_target = tmpl::list<Tags::Square>;
+    using compute_items_on_target = tmpl::list<Tags::NegateComputeItem>;
+    using compute_target_points =
+        intrp::Actions::LineSegment<InterpolationTargetB, 3, Frame::Inertial>;
+    using post_interpolation_callback =
+        TestFunction<InterpolationTargetB, Tags::Negate>;
+    // This tag is also an OptionsTag. The type and help string below
+    // refer to the options that are read from the input file.
+    using type = typename compute_target_points::options_type;
+    static constexpr OptionString help = {"Options for InterpolationTargetB"};
+  };
+
+  using interpolator_source_vars = tmpl::list<gr::Tags::Lapse<DataVector>>;
```

Comment:
make a new tag instead of `Lapse`, because the value being computed isn't really a lapse.